src/modules/io.weldr.anaconda

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+from contextlib import contextmanager
+from types import SimpleNamespace
+import json
+import os
+import subprocess
+import sys
+import tempfile
+
+options = SimpleNamespace(**json.loads(sys.stdin.read()))
+
+with tempfile.NamedTemporaryFile('w', suffix='.ks') as f:
```

Comment:
I try to follow the convention of prefixing lorax's temporary things with lorax-something so that when they inevitably get left laying around the filesystem it's easy to identify them and remove them :)

Reply:
Oh, that's a great idea!